hadoop-connector/com/vertica/hadoop/VerticaConfiguration.java

```
@@ -527,6 +538,15 @@ public String getOutputRecordTerminator() {
 		return conf.get(OUTPUT_TERMINATOR_PROP, RECORD_TERMINATOR);
 	}
 
+  /**
+   * Return the output committer that the job should use
+   * @return the output committer class to use
+   * @throws ClassNotFoundException if the output committer can't be found
+   */
+  public Class getOutputCommitterClass() throws ClassNotFoundException {
```

Does this need to return just the class or could it return an instance? If this is only used to create immediately create an instance of the class you could do the instantiation here and catch ClassCastException etc. so that it doesn't have to be dealt with elsewhere. It would probably also let you fail fast if the class is the wrong type.